app/views/spree/admin/zones/_zip_members.html.erb

```diff
@@ -1,9 +1,9 @@
 <div id="zip_members" data-hook="member">
   <fieldset class="no-border-bottom">
-    <legend align="center"><%= Spree.t(:zipcode) %></legend>
+    <legend align="center"><%= I18n.t(:zipcode) %></legend>
```

Comment:
We can use `t('spree.zipcode')` here, and it will take the translation from the main `.yml` file provided in Solidus core.